examples/common/pigweed/rpc_services/OtCli.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#if CHIP_ENABLE_OPENTHREAD
+#include <openthread/cli.h>
+#include <openthread/instance.h>
+#endif // CHIP_ENABLE_OPENTHREAD
+
+#include "ot_cli_service/ot_cli_service.rpc.pb.h"
+#include "platform/ConfigurationManager.h"
+#include "pw_log/log.h"
+
+namespace chip {
+namespace rpc {
+
+class OtCli : public pw_rpc::nanopb::OtCli::Service<OtCli>
+{
+public:
+    virtual ~OtCli() = default;
+
+    virtual void Command(const ::chip_rpc_OtCliCommand & request, ServerWriter<::chip_rpc_OtCliResponse> & writer)
+    {
+#if CHIP_ENABLE_OPENTHREAD
+#if OPENTHREAD_API_VERSION >= 85
```

Comment:
I think other Matter components require much newer OT API, so this may be redundant, or you can `static_assert` that.